.cloudtest.yaml

```diff
@@ -1,36 +0,0 @@
----
```

Comment:
Do we need to delete these files?  Does it effect anything to keep them?

Reply:
I can add them back in. I just thought it would be easier to understand what's going on if we didn't have unused files lying around.

Reply:
It'll be easiest to maintain the upstream rebases if we stick as much to purely additive changes as possible.  If these files change upstream and we've deleted them in our downstream fork then we'll likely have to resolve the conflicts manually every rebase.

Reply:
I see that makes sense. I can add back in the .cloudtest.yaml file as well I guess, even though the jenkinsfile is not using that one.